KILTprotocol/kilt-extension-api
# refactor: well known did configuration tools

## relates to https://github.com/KILTprotocol/ticket/issues/2126

Uses tooling in the sdk to produce and verify the well-known did configuration. Produces a backwards-compatible but safer proof.

## How to test:

Try integrating in:

- [x] Sporran (for verification) 
    - unfortunately, importing as cjs does not work; I had to change the target to ESM to make it work with sporran 
- [x] SKYC (for generation)

Sporran using the library should also work with the current version of SKYC (not using the library) and vice versa. 

## Checklist:

    * Either PR or Ticket to update [the Docs](https://github.com/KILTprotocol/docs)
    * Link the PR/Ticket here
